### PR TITLE
Map VistA facility information into FHIR format

### PR DESCRIPTION
**Pull request by jbalboni**
## Description
This updates the new appointment flow to map direct scheduling institution data into the FHIR Location resource format. It's mostly test updates, but there are some field and mapping tweaks as well.

## Testing done
Local and unit testing.

## Acceptance criteria
- [ ] App still works, AC in ticket met

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/services/location/index.js [hunk 1]
```diff
@@ -96,3 +102,27 @@ export async function getLocation({ facilityId }) {
...
+/**
+ * Get the parent organization of a given location
+ *
+ * @export
+ * @param {Arrray} organizations The organizations to search through
+ * @param {Object} location The location resource to find the parent of
+ * @returns {Object} The parent organization
+ */
+export function getParentOfLocation(organizations, location) {
+  const orgId = location.managingOrganization.reference.split('/', 2)[1];
```
Comment (narin) [inline src/applications/vaos/services/location/index.js:115]: I don't think  the `2` limit in this split is needed.

File: src/applications/vaos/utils/data.js [hunk 2]
```diff
@@ -42,27 +43,27 @@ export function transformFormToVARequest(state) {
...
   const typeOfCare = getTypeOfCare(data);
-  const parentOrg = getParentFacilities(state).find(
-    parent => parent.id === data.vaParent,
-  );
-  const parentSiteId = getSiteIdFromOrganization(parentOrg);
-  const rootOrg = getRootOrganizationFromChosenParent(state);
-  const rootSiteId = getSiteIdFromOrganization(rootOrg);
+  const parentOrg = getChosenParentInfo(state);
+  // Calling this a facility id instead of a site id because it might be 3 or 5 digits
+  // However, in the future, I believe all of these ids from an Organizaiton will be 3 digits
```
Comment (narin) [inline src/applications/vaos/utils/data.js:48]: Typo in "Organization"

File: src/applications/vaos/utils/selectors.js [hunk 3]
```diff
@@ -12,7 +12,18 @@ import {
...
+import {
+  getRootOrganization,
+  getSiteIdFromOrganization,
+  getIdOfRootOrganization,
+} from '../services/organization';
+import { getParentOfLocation } from '../services/location';
+
+// Only use this when we need to pass data that comes back from one of our
+// services files to one of the older api functions
+function parseFakeFHIRId(id) {
```
Comment (narin) [inline src/applications/vaos/utils/selectors.js:24]: I noticed a similar function in 5 different files.  Would it be worth creating a single util somewhere? I might have mentioned this before, so apologies if you already provided reasoning against it.

**Review by narin — APPROVED** [review]
Some minor feedback but otherwise LGTM.

File: src/applications/vaos/utils/selectors.js (hunk 3 above)
Comment (jbalboni) [reply src/applications/vaos/utils/selectors.js:24]: Heh, my reasoning is that these are temporary and I want them to standout as bad/weird code so no one thinks they're supposed to be commonly used. All uses of them will go away when we're through all the mapping and update tickets. I'm removing a bunch of them in my in progress ticket, too.

**Pull request merged**